src/renderer/components/ConnectionsProtocol.js

```diff
@@ -70,9 +70,13 @@ export default {
         }
       };
       console.log("Exchange Request", req);
+      const signed = didcomm.signedAttachment(req.connection.DIDDoc, toolbox_did)
+      console.log("Example attachment signing:", signed)
+      console.log("Example attachment verification:", didcomm.verifySignedAttachment(signed))
+      console.log("Example attachment decoding:", didcomm.decodeSignedAttachment(signed))
```

Comment:
These lines are of particular note as they demonstrate how the newly added methods are to be used. Observe the output in your console on newly created connections with the toolbox.